src/views/open-api-docs-view/server.ts

```diff
@@ -96,6 +107,7 @@ export async function getStaticProps({
 				targetVersion,
 				schemaData,
 			},
+			operationGroups: stripUndefinedProperties(operationGroups),
```

Comment:
I've updated the server function to parse & return the `operationGroups` props needed for scaffolding.